package.json

```diff
@@ -1,8 +1,11 @@
 {
   "scripts": {
-    "format": "prettier --write ."
+    "format:check": "npx prettier --check \"**/*.md\"",
```

Comment:
Suggerimento: Non utilizzerei `npx` per avviare Prettier avendolo già installato nelle dipendenze.

Reply:
Ok, credo che sia stato inserito per evitare di installare le `devDeps` direttamente dalla action. È strettamente necessario secondo voi? 

Personalmente "normalizzerei" gli scripts in maniera tale che vengano tutti avviati in un'unica maniera.

Reply:
Al momento è l'unica action che abbiamo e che ha bisogno di `node`.

Volendo si può rimuovere `node` e lanciare il comando a mano, ma poi se si cambia nel package.json andrebbe cambiato anche nella actions.

Sinceramente non installerei le deps e non le metterei nemmeno in cache, o almeno, non al momento.